### DISCUSSION TITLE
Problems getting EX-T3 working

### DISCUSSION THREAD

**Discussion by plingboot** [DCC-EX/EX-T3-WiFi#77] [Q&A]
I've completed assembly - minus the battery pack (on order) and power switch. I'm testing it via USB power (which I assume should work) but all that's happening is the screen flashing.
Just wondered whether any has any helpful suggestions. Attached clip shows what's happening and how it's all connected up.
Thanks.
https://user-images.githubusercontent.com/452962/226295964-fd323e8a-585b-41e8-a9b5-5d63e9856c13.mov

**Comment by alex-code** [comment]
Did you upload the code and the filesystem?
Currently there's 2 separate uploads that need to be done.

- File System - On the PlatformIO tasks pane, `firebeetle2` -> `Platform` -> `Upload Filesystem Image`
- Code - On the PlatformIO tasks pane, `firebeetle2` -> `General` -> either `Upload` or `Upload and Monitor`

**Reply by plingboot** [reply]
That did it - I'd not uploaded the file system image task. It's just connected to my Mac at the moment, but looks like it's working. Will give it a test with my DIY DCC-EX station later. Thanks again. :)

**Reply by plingboot** [reply]
Alex, if you check out plingboot on Instagram, I have a short clip of the remote in action. 🙂
